pkg/source/examples/simple_source/Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: build
+build:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -o ./dist/simple-source-example main.go
+
+.PHONY: image
+image: build
+	docker build -t "quay.io/numaio/numaflow-go/source-simple-source:latest" --target simple-source .
```

```suggestion
	docker build -t "quay.io/numaio/numaflow-go/source-simple-source:v1" --target simple-source .
```

so we can version images and that way e2e tests won't break?